### PR TITLE
Some dynamic registry mappings

### PR DESCRIPTION
**Pull request by williewillus**
When you have the DFU hammer, everything starts looking like a nail to Mojang.

### CODE REVIEW

**Review by Gegy — APPROVED** [review]

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/util/dynamic/RegistryOps.mapping [hunk 1]
```diff
@@ -17,12 +19,28 @@ CLASS net/minecraft/class_5382 net/minecraft/util/dynamic/RegistryOps
...
 	CLASS class_5383 ValueHolder
 		FIELD field_25513 values Ljava/util/Map;
+	CLASS class_5506 EntryLoader
+		METHOD method_31154 resourceBacked (Lnet/minecraft/class_3300;)Lnet/minecraft/class_5382$class_5506;
+		METHOD method_31155 load (Lcom/mojang/serialization/DynamicOps;Lnet/minecraft/class_5321;Lnet/minecraft/class_5321;Lcom/mojang/serialization/Decoder;)Lcom/mojang/serialization/DataResult;
+			ARG 2 registryId
+			ARG 3 entryId
+		METHOD method_31156 getKnownEntryPaths (Lnet/minecraft/class_5321;)Ljava/util/Collection;
+			COMMENT @return A collection of file Identifiers of all known entries of the given registry.
+			COMMENT                 Note that these are file Identifiers for use in a resource manager, not the logical names of the entries.
```
Comment (YanisBft) [inline mappings/net/minecraft/util/dynamic/RegistryOps.mapping]: Weird indentation here

Comment (williewillus) [reply mappings/net/minecraft/util/dynamic/RegistryOps.mapping]: I was trying to make it match the previous line (so it looks correct in IDE's). Should I not do that/will it be correctly formatted when applied?

Comment (YanisBft) [reply mappings/net/minecraft/util/dynamic/RegistryOps.mapping]: Would be better to start a new paragraph instead of trying this

Comment (liach) [reply mappings/net/minecraft/util/dynamic/RegistryOps.mapping]: yeah, consider writing these as method doc than return tag content

**Comment by YanisBft** [comment]
@williewillus Could you update this and resolve the comment please?

**Comment by williewillus** [comment]
fixed and rebased

**Review by YanisBft — APPROVED** [review]

**Pull request merged**